Runners/WebRunner/Controllers/WhatsappBotController.cs

```diff
@@ -20,7 +20,7 @@ public class WhatsappBotController : TwilioController
 
         public WhatsappBotController(PluginExecutor pluginExecutor)
         {
-            _pluginExecutor ??= pluginExecutor;
+            _pluginExecutor = pluginExecutor;
         }
```

Comment:
Maybe also do the same to the other controllers

Reply:
It is the same in other controllers